boot/setup.sh

```diff
@@ -94,6 +98,8 @@ function setup_buckets_mdapi {
 
     svccfg import /opt/smartdc/buckets-mdapi/smf/manifests/buckets-mdapi-setup.xml
     svccfg import /opt/smartdc/buckets-mdapi/smf/manifests/buckets-mdapi.xml
+
+    mdata-put metricPorts $(echo "${metrics_port}")
```

Comment:
This can be simplified to:
`mdata-put metricPorts "${metrics_port}"`

Reply:
Yes makes sense. Done!